packages/@icon-magic/generate/src/plugins/svg-generate.ts

```diff
@@ -54,12 +59,34 @@ export const svgGenerate: GeneratePlugin = {
   fn: async (
     flavor: Flavor,
     icon: Icon,
-    params: SvgGenerateOptions = {}
+    params: SvgGenerateOptions = {},
+    hashing?: boolean
```

Comment:
Why does the plugin need to be aware of hashing? Can we skip applying the plugins if hashing is true?

Reply:
Thought about that, if the plugin is unaware of hashing then it's either all or nothing for an icon config. i.e if we have

```
  nav-icons
  ├── home
  │   ├── solid.svg
  │   └── color.svg
  ├── iconrc.json
```

And we have a new variant `mixed.svg` dropped in. Either it will take all of them through the process or none (including the new one). This way, hashing can be true and the plugin only run for the new addition `mixed.svg`.

Same thing if webps have been generated but svgs have not been minified. Skipping the plugings completely would mean they both have to be run if the user sets hashing to false. 

So depends on what is important to us.

Reply:
also the `assetName` for `svg-to-raster` for example is computed within the plugin and we need that to check it's existence in the iconrc.

Reply:
If there's value in passing it to the plugin, we should still keep the plugin interface the same both for build and generate. The problem I have here is the inconsistency between the two.